### ISSUE TITLE
VSCode Error on latest version

### ISSUE THREAD

**Issue by chris-crowson** [RimuruChan/vscode-fix-checksums#3]
VSCode changed how they do ESM loading in the latest version - https://github.com/microsoft/vscode-docs/blob/vnext/release-notes/v1_93.md#impact-of-ecmascript-module-esm-loading-of-vs-code.

This introduced a bug on line https://github.com/RimuruChan/vscode-fix-checksums/blob/d394c4cd826df5b58d1d6e80dc142f6148f63c41/extension.js#L6.

It needs to be updated to now read:
`const appDir = `${path.dirname(vscode.env.appRoot)}/app/out`;`

**Comment by ArcticOc** [comment]
Same issue. It seems that this update `const appDir = ${path.dirname(vscode.env.appRoot)}/app/out;` doesn’t make the extension work properly.

**Comment by gitiy1** [comment]
```
Version: 1.94.0
Commit: d78a74bcdfad14d5d3b1b782f87255d802b57511
Date: 2024-10-08T21:35:30.091Z
Electron: 30.5.1
ElectronBuildId: undefined
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Linux x64 6.11.2-zen1-1-zen
```

I am on Code - OSS installed from pacman Extra on Arch Linux with KDE Desktop Environment 6.1.5 Wayland and the update worked for me is 

```js
const appDir = path.join(vscode.env.appRoot, 'out');
```

and if i use the Master branch and run fixChecksums.apply, it seems like the process.execPath could only return the Electron Path which is `/usr/lib/Electron30` and with no doubt it couldn't work and throw an errror


The app launch command is

```bash
code-oss %F
```

and the flags are

```bash
--ozone-platform-hint=auto
--enable-wayland-ime
--use-vulkan
--enable-gpu-rasterization
--enable-native-gpu-memory-buffers
--enable-quic
--enable-zero-copy
--gpu-rasterization-msaa-sample-count=16
--ignore-gpu-blocklist
--num-raster-threads=4
--ozone-platform-hint=auto
--enable-smooth-scrolling
--webgl-msaa-sample-count=16
--enable-features=UseOzonePlatform,Vulkan,EnableTLS13EarlyData,CompressionDictionaryTransport,CompressionDictionaryTransportBackend,DefaultANGLEVulkan,FluentOverlayScrollbar,FluentScrollbar,OverlayScrollbar,ParallelDownloading,PostQuantumKyber,UIEnableSharedImageCacheForGpu,UseGpuSchedulerDfs,SharedZstd,ZstdContentEncoding
```

I am not sure if this was cause by code-oss or Linux env logic, it seems like vscode.env.appRoot only returns `/usr/lib` and i am not sure if this problem exists with Code Server or VSCodium Users and on other platforms

**Comment by RimuruChan** [comment]
Fixed in #4
